tests/test_electrolyzer.py

```diff
@@ -151,6 +155,19 @@ def test_electrolyzer_inequality(self):
                 results["Electrolyzer_fc66.ElectricityIn.Power"] * a[i] + b[i] + 1.0e-3,
             )
 
+        # Check electrolyzer input power
+        np.testing.assert_allclose(
+            results["Electrolyzer_fc66.ElectricityIn.Power"],
+            [1.00000000e08, 1.00000000e08, 1.00000000e08],
+            atol=1e-4,
```

Comment:
Furthermore, what is the reason for the specific absolute tolerance of 1e-4?

Reply:
The bound is 5e8W